tests/modules/modules_json.py

```diff
@@ -62,6 +64,57 @@ def test_mod_json_create(self):
         assert "branch" in mod_json["repos"][NF_CORE_MODULES_NAME]["modules"][mod]
 
 
+def modify_main_nf(path):
+    """Modify a file to test patch creation"""
+    with open(path, "r") as fh:
+        lines = fh.readlines()
+    # Modify $meat.id by $meta.single_end
+    lines[1] = '    tag "$meta.single_end"\n'
+    with open(path, "w") as fh:
+        fh.writelines(lines)
+
+
+def test_mod_json_create_with_patch(self):
+    """Test creating a modules.json file from scratch when there are patched modules"""
+    mod_json_path = Path(self.pipeline_dir, "modules.json")
+
+    # Modify the module
+    module_path = Path(self.pipeline_dir, "modules", "nf-core", "modules", "fastqc")
+    modify_main_nf(module_path / "main.nf")
+
+    # Try creating a patch file
+    patch_obj = ModulePatch(self.pipeline_dir)
+    patch_obj.patch("fastqc")
+
+    patch_fn = "fastqc.diff"
+    # TO REMOVE
```

Comment:
```suggestion
```
Not sure about this line 🙂

Reply:
Oops, thanks for spotting it! I wanted to remove the following line as the creation of the patch file is already tested in other tests.